tests/bsim/bluetooth/host/privacy/peripheral/src/tester_rpa_expired.c

```diff
@@ -22,6 +31,14 @@ struct adv_set_data_t {
 static	uint8_t	adv_index;
 static	struct	adv_set_data_t	adv_set_data[CONFIG_BT_EXT_ADV_MAX_ADV_SET];
 
+static void print_address(bt_addr_le_t *addr)
```

Comment:
It seems a bit silly to copy this around.. don't you want to add it to the babblekit library?
(or leave it in that bs_bt_utils.c as a more evident thing somebody will be inclined to clean later :) )

Reply:
I agree, but I don't think it belongs in the babblekit library as it doesn't do anything bsim related. Any idea of where else to put it?
(I'd rather not keep bs_bt_utils.c/h only for this function)

Reply:
You could put it in the testlib maybe (`tests/bluetooth/common/testlib`)

Reply:
Could we perhaps instead pull in `bt_addr_le_str` from https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/common/bt_str.c ?

Reply:
I added a macro to testlib (addr.h) which allows you to print multiple addresses in the same printk/LOG_DBG etc. (bt_addr_le_str will print the same address if used multiple times in a format string)